### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by wjakelee**
Adding some additional context and examples for python instrumentation

## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/index.md [hunk 1]
```diff
@@ -92,6 +92,16 @@ When everything is configured it is very simple to run an instrumented applicat
 opentelemetry-instrument python3 SCRIPT_NAME.py
 ```
 
+For Auto-instrumentation to work, you will need to include `opentelemetry-instrument` at the beginning of your application execution. If this is not an option for you, then you will need to follow the usage steps for each instrumentation library you install. For example, for Django instrumentation, you will need to add the following two lines to your code (outlined in the [Usage section](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/django/django.html#id1) of the library documentation):
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/index.md:95]:
```suggestion
To enable auto-instrumentation, you'll need to include `opentelemetry-instrument` at the beginning of your application execution. If this is not an option for you, you’ll need to follow the usage instructions for each instrumentation library you install.  For example, for Django instrumentation, you will need to add the following two lines to your code (outlined in the [Usage section](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/django/django.html#id1) of the library documentation):
```

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/index.md [hunk 2]
```diff
@@ -92,6 +92,16 @@ When everything is configured it is very simple to run an instrumented applicat
...
+
+```
+from opentelemetry.instrumentation.django import DjangoInstrumentor
+
+DjangoInstrumentor().instrument()     ##This goes inside of main
+```
+
+Adding these lines prevents you from having to use the `opentelemetry-instrument` command in the execution of your app.
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/index.md:103]:
```suggestion
Adding these lines eliminates the need to use the `opentelemetry-instrument` command in the execution of your app.
```

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/index.md [hunk 3]
```diff
@@ -160,3 +170,15 @@ By default, OpenTelemetry uses W3C context propagation standard. If application
 ## Log correlation
 
 * To enable trace context injection into logs, set the `OTEL_PYTHON_LOG_CORRELATION` environment variable to `true` ([learn more](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/logging/logging.html#enable-trace-context-injection))
+
+
+## Examples
+
+### Django Instrumentation
+
+[This article](https://opentelemetry-python.readthedocs.io/en/stable/examples/django/README.html#django-instrumentation) will show you how to use the `opentelemetry-instrumentation-django` library to automatically instrument a Django app. It will show you how to reference the library manually within your code, as well as how to use [Auto-instrumentation](https://opentelemetry-python.readthedocs.io/en/stable/examples/django/README.html#auto-instrumentation) without having to edit your code.
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/index.md:179]:
```suggestion
[This article](https://opentelemetry-python.readthedocs.io/en/stable/examples/django/README.html#django-instrumentation) will show you how to use the `opentelemetry-instrumentation-django` library to automatically instrument a Django app. It covers how to reference the library manually within your code, as well as how to use [Auto-instrumentation](https://opentelemetry-python.readthedocs.io/en/stable/examples/django/README.html#auto-instrumentation) without having to edit your code.
```

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/index.md [hunk 4]
```diff
@@ -160,3 +170,15 @@ By default, OpenTelemetry uses W3C context propagation standard. If application
...
+
+### Sample Flask App Instrumentation
+
+[This article](https://opentelemetry.io/docs/languages/python/getting-started/) will walk you through deploying your own simple Flask app and configuring instrumentation on that app. The example will log the traces to the console for simple testing.
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/index.md:183]:
```suggestion
[This article](https://opentelemetry.io/docs/languages/python/getting-started/) walks you through deploying a simple Flask app and configuring instrumentation for it. The example demonstrates how to log traces to the console for basic testing.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Looks great! I noted some suggestions.

**Comment by wjakelee** [comment]
Love it, thank you!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**